### ISSUE TITLE
create term for 'increased outer canthal distance'

### ISSUE THREAD

**Issue by azankl** [obophenotype/human-phenotype-ontology#11494]
**Preferred term label:**
increased outer canthal distance

**Synonyms**

**Definition (free text, please give PubMed ID)**
increased distance (more than 97th centile or 2SD) between the outer canthi of the eyes

**Parent term (use hpo.jax.org/app)**
Abnormality of globe location HP:0100886

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
many, eg. Aarskog-Scott syndrome (https://pubmed.ncbi.nlm.nih.gov/14560308/)

**Your nano-attribution (ORCID)**
Andreas Zankl 0000-0001-8612-1062
